lightningd/invoice.c

```diff
@@ -976,15 +984,15 @@ static void listincoming_done(const char *buffer,
 }
 
 void invoice_check_onchain_payment(struct lightningd *ld,
-					  const u8 *scriptPubKey, struct amount_sat sat, struct bitcoin_outpoint outpoint)
+					  const u8 *scriptPubKey, struct amount_sat sat, struct bitcoin_outpoint *outpoint STEALS)
```

Comment:
STEALS is always ugly, and in this case it's better for this to make a copy if it wants.  Also, `const struct...`